src/App.css

```diff
@@ -1,42 +1,63 @@
 #root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
+  max-width: 100%;
+ 
+
   text-align: center;
 }
 
-.logo {
-  height: 6em;
-  padding: 1.5em;
-  will-change: filter;
-  transition: filter 300ms;
+
+
+
+.article-list{
+  margin: 0;
+  background-color: #284B63;
+  width: 98vw;
+  
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: space-around;
 }
-.logo:hover {
-  filter: drop-shadow(0 0 2em #646cffaa);
+
+@media only screen and (max-width: 600px){
```

Comment:
Ooooo, love the responsiveness, unfortunately, I can't read the text on the small view very well.